CONTRIBUTING.md

```diff
@@ -0,0 +1,235 @@
+# Contributing
+
+The following documentation is only for the maintainers of this repository.
+
+- [Monorepo setup](#monorepo-setup)
+- [Installation](#installation)
+- [Develop the CLI packages](#Develop-the-CLI-packages)
+- [Release the packages](#release-the-packages)
+- [Available commands](#commands)
+- [CI](#ci)
+- [Add a new package to the monorepo](#add-a-new-package-to-the-monorepo)
+
+## Monorepo setup
+
+This repository is managed as a monorepo that is composed of many npm packages.
+
+For more information on monorepo:
+
+- [Babel GitHub](https://github.com/babel/babel/blob/master/doc/design/monorepo.md)
+- [Shopify GitHub](https://github.com/Shopify/quilt/blob/master/Decision%20records/00%20-%20Use%20a%20Lerna%20monorepo.md)
+- [Google](https://www.google.com/search?q=monorepo)
+
+### PNPM workspace
+
+This monorepo is using PNPM workspace feature to handle the installation of the npm dependencies and manage the packages interdependencies.
+
+It's important to note that PNPM workspace will **hoist** the npm dependencies at the root of the workspace. This means that there might not be a *node_modules* directory nested in the packages directories. The npm dependencies are installed in a *node_modules* directory at the root of the workspace and a single *pnpm-lock.yaml* file is generated at the root of the workspace.
```

Comment:
In fact, PNPM doesn't "hoist" dependencies, this is one of the nice thing about PNPM :)

Reply:
Yea, `link` is prbably a better choice.